### PR TITLE
feat: add some convenience redirects

### PR DESCRIPTION
**Pull request by feedm3**
Every now and then I manually enter the URL to one of our tools. My browser always autocompletes to `satellyes.com`, but never to `slack.satellytes.com` (too many different URLs with `slack` in the title or history). Same with the other URLs. 

With those redirects, the autocomplete in the browser gets more useful.

Test:
- https://satellytescommain21751-featconvenienceredirects.gtsb.io/github

### CODE REVIEW

File: gatsby/create-pages/create-redirects.ts [hunk 1]
```diff
@@ -156,16 +180,25 @@ export const createRedirects = async ({ actions }: CreatePagesArgs) => {
...
       toPath: redirect.toPath,
       isPermanent: true,
       redirectInBrowser: true,
     });
   });
+
+  EXTERNAL_REDIRECTS.forEach((redirect) => {
+    createRedirect({
+      fromPath: redirect.fromPath,
+      toPath: redirect.toUrl,
```
Comment (feedm3) [inline gatsby/create-pages/create-redirects.ts:199]: On my local machine, the redirect to an external domain was working - to my surprise. I always thought that external URLs don't work. Pretty sure it also works on Gatsby Cloud then.

**Review by georgiee — COMMENTED** [review]
That's convenient indeed 🙌 I'm worrying about google indexing those. I think we should prevent that as I don't want that those urls appear in google for any reason. How would we put those urls on a robots list or similar? Or is that exactly risky because we expose specifically our "secret" urls ? 😄

This is mixing internal and external interests which makes me uncomfortable. Is there any risk or issue beyond google doing that? Maybe we should wait for some other opinions on that 🙏

**Comment by feedm3** [comment]
We can use the `x-robots-tag: noindex` header to prevent indexing the URLs: 

https://developers.google.com/search/docs/advanced/robots/robots_meta_tag#xrobotstag

I will put the code into #615 where we have control over all headers.

**Comment by georgiee** [comment]
Thank you for the clarification and the idea to put it into the 404 PR🙌

**Pull request closed without merging**